src/storage_persister.rs

```diff
@@ -25,9 +39,33 @@ impl StoragePersister {
         self.storage.check_health(MONITORS_BUCKET.to_string())
     }
 
+    pub fn check_object_storage_health(&self) -> bool {
+        self.storage.check_health(OBJECT_BUCKET.to_string())
+    }
+
     pub fn read_channel_monitors(&self) {
         // TODO: Implement
     }
+
+    pub fn read_channel_manager(&self) {
+        // TODO: Implement
+    }
+
+    pub fn read_graph(&self) {
+        // TODO: Implement
+    }
+
+    pub fn read_scorer(&self) {
+        // TODO: Implement
+    }
+
+    fn persist_object(&self, bucket: String, key: String, data: Vec<u8>) -> Result<(), Error> {
+        if !self.storage.put_object(bucket, key, data) {
+            // Operating System I/O Error
+            return Err(Error::from_raw_os_error(5));
```

Comment:
Can we explicitly create `Error` with a value from [`ErrorKind` enum](https://doc.rust-lang.org/std/io/enum.ErrorKind.html) and a message (using `Error::new()` or `Error::other()` method)?